code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -663,20 +667,39 @@
 
 				else
 					if(brutedamage > 0)
-						status = "bruised"
-					if(brutedamage > 20)
-						status = "battered"
-					if(brutedamage > 40)
-						status = "mangled"
+						if(is_robotic_limb)
+							status = "marred"
+						else
+							status = "bruised"
+					if(brutedamage > (limb_max_damage*0.4))
+						if(is_robotic_limb)
+							status = "dented"
+						else
+							status = "battered"

```

Comment:
Maybe these should be vars on the limbs so if we add more later we don't need a stack of if elses?